static/js/hero.js

```diff
@@ -46,59 +46,52 @@ $(document).ready(function() {
   // its existence before we try anything.
   $("#actionButton").on("click", function(event) {
     event.preventDefault();
-
-    const hlsAboutVideo = $("#tv-light-box-video").get(0);
-    const aboutVideoYoutube = $("#tv-yt-light-box-video")
-      .find("iframe")
-      .get(0);
-
-    if (hlsAboutVideo) {
-      showLightBox();
-      hlsAboutVideo.play();
-    } else if (aboutVideoYoutube) {
-      showLightBox();
-      aboutVideoYoutube.contentWindow.postMessage(
-        JSON.stringify({
-          event: "command",
-          func:  "playVideo"
-        }),
-        "https://www.youtube.com"
-      );
+    const youtubeVideoSrc = $("#tv-light-box-yt-video").attr("data-href");
+    const hlsAboutVideoSrc = $("video#tv-light-box-video");
+    let backgroundVideo = null;
+    if (youtubeVideoSrc) {
+      $.fancybox.open({
+        src:        youtubeVideoSrc,
+        beforeShow: function() {
+          backgroundVideo = $("#background-video").get(0);
+        },
+        beforeLoad: function() {
+          pauseBackgroundVideo(backgroundVideo);
+        },
+        afterClose: function() {
+          resumeBackgroundVideo(backgroundVideo);
+        }
+      });
+    } else if (hlsAboutVideoSrc) {
+      $.fancybox.open({
+        content:    hlsAboutVideoSrc,
+        type:       "html",
+        beforeShow: function() {
+          backgroundVideo = $("#background-video").get(0);
+        },
+        beforeLoad: function() {
+          pauseBackgroundVideo(backgroundVideo);
+        },
+        afterClose: function() {
+          resumeBackgroundVideo(backgroundVideo);
+        }
+      });
     } else {
       console.error("We do not have any supported video elements available."); // eslint-disable-line no-console
     }
   });
 });
 
-function closeLightBox() {
-  // Closes the light box.
-
-  const hlsAboutVideo = $("#tv-light-box-video").get(0);
-  const aboutVideoYoutube = $("#tv-yt-light-box-video")
-    .find("iframe")
-    .get(0);
-
-  if (hlsAboutVideo) {
-    hlsAboutVideo.pause();
+function resumeBackgroundVideo(backgroundVideo) {
+  // Resume the background video of hero section.
+  if (backgroundVideo) {
+    backgroundVideo.play();
```

Comment:
I don't think we need separate functions defined for `resumeBackgroundVideo` and `pauseBackgroundVideo`. I think it's fine just to define them in the `beforeLoad` and `afterClose` anonymous functions

```javascript
    afterClose: function() {
      backgroundVideo && backgroundVideo.play();
    }
```